loguru/_logger.py

```diff
@@ -1752,6 +1754,46 @@ def configure(self, *, handlers=None, levels=None, extra=None, patcher=None, act
 
         return [self.add(**params) for params in handlers]
 
+    def _replace_core(self, core: Core):
+        self._core = core
+
+    def reinstall(self):
+        """Reinstall the core of logger.
+
+        When using multiprocessing, you can pass logger as a parameter to the target of
+        ``multiprocessing.Process``, and run this method once, thus you don't need to pass logger to every
+        function you called in the same process with spawn multiprocessing.
+
+        Examples
+        --------
+        >>> def subworker(logger_):
+        ...     logger_.reinstall()
```

Comment:
I think it would be more convenient to name the argument `logger` instead of `logger_`. Then we can do:

```python
logger.reinstall()
logger.info("Child")
```

It should work transparently.